express/package.json

```diff
@@ -18,36 +18,37 @@
   },
   "dependencies": {
     "@aws-sdk/client-cognito-identity-provider": "^3.80.0",
+    "@aws-sdk/client-sns": "^3.162.0",
```

Comment:
The express app doesn't need to have this package, or any other new packages, so the changes to this file should be reverted.